blog/2020-05-08-AIVersusCovid-19.adoc

```diff
@@ -0,0 +1,135 @@
+this is a soft constraint.
+
+* The higher an employee's Covid-19 susceptibility is, the less they should
+work in a Covid-19 ward. This is another soft constraint.
+
+These, and the rest of the constraints, can be found in the
+https://github.com/kiegroup/optaweb-employee-rostering/blob/covid-19/optaweb-employee-rostering-backend/src/main/resources/org/optaweb/employeerostering/service/solver/employeeRosteringScoreRules.drl[Drools constraints
+file]. They can also be implemented in Java using constraint streams.
+
+== Using the application
+
+OptaWeb Employee Rostering comes with both a backend and frontend. You could use the
+bundled frontend or use the backend's REST endpoints. The application by default
+starts with demo data, allowing you to try out the application. Let's use one of
+example tenant and solve its roster. Navigate to the "Scheduling" tab and click
+the (?) button next to the score. You will see a detailed summary of all the constraints.
+In it, we see the constraint "Assign every shift" has 1116 matches, meaning we have
+1116 unassigned shifts. Let's fix that right now.
+
+image::ConstraintSummary1.png[Unassigned Shifts]
+
+Close the summary popup and click on the "Solve" button. OptaPlanner will begin solving
+the roster. After about 30 seconds, you'll see there are no more unassigned shifts --
+that is, OptaPlanner assigned an employee to each of the 1116 shifts, all the while
+making sure no employee get two shifts at the same time, each employee has the required
+skills for their shift, and each employee has enough rest between shifts. Imagine
+making a feasible schedule by assigning 1116 shifts by hand; it would take a good
+while longer than 30 seconds!
+
+image::ConstraintSummary1.png[Assigned Shifts]
+
+While solving, OptaPlanner will keep looking for better solutions, gradually improving
+the score. You might have already seen the schedule change several times already.
+Let's stop solving for now. Click the Ward selector (left side of the toolbar) and
+change to the ward "COVID-19 Ward 1". Every shift has a small circle at the top of
+it denoting the Covid-19 susceptibility of its employee. A shield represents
+inoculated employees; people who have acquired immunity to Covid-19. A near empty
+circle mean the employee has a low Covid-19 susceptibility, while a more filled in
+circle mean a higher Covid-19 susceptibility. Finally, a red circle mean the employee
+is extremely susceptible to Covid-19, and should not placed in a ward with Covid-19.
+Notice how there are no Extreme risk employees in the Covid-19 ward? Moreover, most
+of the Covid-19's ward shifts either have inoculated or low risk employees.
+
+image::CovidWardNoExtremeRisk.png[No Extreme Risk Employees in Covid-19 Ward]
+
+One second, I am getting a call. Okay, I'm back. Beth P. Watt caught the flu and
+as such is not able to make their shift this Saturday. Luckily for us, OptaPlanner
```

Comment:
```suggestion
as such is not able to make her shift this Saturday. Luckily for us, OptaPlanner
```

Soft suggestion. AFAIK "their" can be used if the gender of the subject is uncertain, e.g. when speaking about "the user". In this sentence it sounds a bit off since Beth is clearly a female name.